synapse/handlers/__init__.py

```diff
@@ -31,10 +31,22 @@
 
 class Handlers(object):
 
-    """ A collection of all the event handlers.
+    """
```

Comment:
It'd be nice if this had a non-empty summary line (http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)